modules/tinymce/src/plugins/lists/test/ts/browser/IndentTest.ts

```diff
@@ -449,6 +449,79 @@ describe('browser.tinymce.plugins.lists.IndentTest', () => {
     assert.equal(editor.selection.getNode().nodeName, 'LI');
   });
 
+  it('TINY-10213: Indent an LI with a list inside should not delete the rest of the content', () => {
+    const editor = hook.editor();
+    editor.setContent(
+      '<ol>' +
+      '<li>a</li>' +
+      '<!-- c1 -->' +
+      '<li>' +
+      '<ol>' +
+      '<li>b</li>' +
+      '</ol>' +
+      '<p data-fake-attr="something">p1</p>' +
+      '<p data-fake-attr="something">p2</p>' +
+      '</li>' +
+      '<!-- c2 -->' +
+      '</ol>'
+    );
+
+    LegacyUnit.setSelection(editor, 'li', 0);
+    editor.execCommand('Outdent');
+
+    TinyAssertions.assertContent(editor,
+      '<p>a</p>' +
```

Comment:
How this paragraph ends up converted from the LI?

Reply:
here the action is `'Outdent'` not `'Indent'`, you can try in fiddle:
```
<ol>
  <li>a</li>
  <li>
    <ol>
      <li>b</li>
    </ol>
  </li>
</ol>
```
and press `shift+tab` (or click on the `toolbar`'s button) in the first `li` since it is at the first level it will be go out of the table

Reply:
Right, turns out CK editor behaves the same way. I expected nothing would happen if there was no room for outdenting.